ca/ca.go

```diff
@@ -720,6 +734,7 @@ type orphanedCert struct {
 	DER      []byte
 	OCSPResp []byte
 	RegID    int64
+	Precert  bool
```

Comment:
Have you done any testing of how `goque` handles things already stored that don't have a field that is present on the object you are casting to? I suspect it'll just silently not fill the field, but it's entirely possible it'll be strict and cause a problem.

Reply:
Good catch, thanks for pointing this out. I changed the `IssueCertificate` path so it leaves the `Precert` field at its zero value, so we get coverage of that case in the tests.

Reply:
Although thinking about this again, this probably leads `goque` to serialize something with `Precert` explicitly set to false, so I'd have to write to the queue with a CA from `master`, shut down, switch to this branch, and bring up the CA again. I'll try that.

Reply:
Okay, I've confirmed that if an orphanedCert is serialized onto the queue and then deserialized into an `orphanedPrecert` struct that has the added field `Precert bool`, deserialization happens correctly and doesn't error. Thanks for checking on this.

Reply:
:+1: thanks.